src/MinedMap.cpp

```diff
@@ -206,8 +206,10 @@ static int64_t getModTime(const std::string &file) {
 		return INT64_MIN;
 	}
 
-#ifdef _WIN32
+#if defined(_WIN32)
 	return (int64_t)s.st_mtime * 1000000;
+#elif defined(__APPLE__) || defined(__NetBSD__)
```

Comment:
According to https://man.netbsd.org/stat.2, these fields are available on NetBSD only when _NETBSD_SOURCE is defined.